dizoo/league_demo/league_demo_ppo_main.py

```diff
@@ -120,18 +121,32 @@ def main(cfg, seed=0, max_iterations=int(1e10)):
 
     for run_iter in range(max_iterations):
         if evaluator1.should_eval(main_learner.train_iter):
-            stop_flag1, reward = evaluator1.eval(
+            stop_flag1, reward, episode_info = evaluator1.eval(
                 main_learner.save_checkpoint, main_learner.train_iter, main_collector.envstep
             )
+            win_loss_result = [e['result'] for e in episode_info[0]]
+            # set fixed policy trueskill equal 0
+            main_player.rating = league.metric_env.rate_1vsC(
+                main_player.rating, league.metric_env.create_rating(mu=0), win_loss_result
```

Comment:
TrueSkill of this fixed policy is unknown. It will be hard to come up with a constant TrueSkill at the beginning. There will be 2 solutions here.
(1) Don't try to assign a constant TrueSkill initially. Make it not constant and can be updated each time after the battle.
(2) Assign a constant TrueSkill initially by the following methods. As we know that NE is optimal so naturally there will be a ceiling for TrueSkill. Change this policy to NE policy, assign TrueSkill 20 and keep it constant. 

Feel free to discuss which one to choose. First one might be better.

Reply:
- I think (2) can be better, random policy is floor while NE policy is ceiling. But if we set random policy to 0, we can't set NE policy just to an arbitrary larger value like 20, we can calculate trueskill of NE policy with respect to random policy.
- Initial policy of main player can be another eval policy, whose weights are random initialized. And the trueskill of it can be initialized as 0 and updated during training procedure.

Reply:
> we can calculate TrueSkill of NE policy with respect to random policy.
Looks good.

> Initial policy of main player can be another eval policy, whose weights are random initialized. And the TrueSkill of it can be initialized as 0 and updated during training procedure.
Looks good too.